examples/supabase/src/components/BarChart.js

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import cubejs from "@cubejs-client/core";
+import { QueryRenderer } from "@cubejs-client/react";
+import { Spin } from "antd";
+import {
+  CartesianGrid,
+  XAxis,
+  YAxis,
+  Tooltip,
+  ResponsiveContainer,
+  Legend,
+  BarChart,
+  Bar
+} from "recharts";
+
+const CartesianChart = ({ resultSet, children, ChartComponent }) => (
+  <ResponsiveContainer width="100%" height={350}>
+    <ChartComponent data={resultSet.chartPivot()}>
+      <XAxis dataKey="x" />
+      <YAxis />
+      <CartesianGrid />
+      {children}
+      <Legend />
+      <Tooltip />
+    </ChartComponent>
+  </ResponsiveContainer>
+);
+
+const cubejsApi = cubejs(
```

Comment:
Can we use `CubeProvider` and initialize API once in `App.js` or `index.js` instead?

Reply:
Here is example - https://cube.dev/docs/@cubejs-client-react

Reply:
updated in new commit